client/src/components/applicants/Applicants.js

```diff
@@ -62,6 +64,18 @@ export class Applicants extends Component {
     }
   }
 
+  toggleShowModal = (applicant) => () => {
```

Comment:
Hyppiikö rivit silmissä, vai onko tässä päällekkäin kaksi samaa funkkaria? Ja nyt tuossa on kahdet propsi sulut peräkkäin, eli `toggleShowModal = (applicant) => () => {` voisi muuttaa muotoon `toggleShowModal = (applicant) => {`